buildtest/cli/compilers.py

```diff
@@ -34,6 +40,42 @@ def compiler_cmd(args, configuration):
         bc.print_yaml()
 
 
+def compiler_test(args, configuration):
+    """This method implements ``buildtest config compilers test`` which tests
+    the compilers with the corresponding modules if set. This command iterates
+    over all compilers and perform the module load test and show an output of
+    each compiler with a PASS or FAIL next to each compiler section.
+    """
+    bc = BuildtestCompilers(debug=args.debug, configuration=configuration)
+    bc.find_compilers()
+    console = Console()
+
+    table = Table(title="Compilers Test Pass")
+    table.add_column("No.", style="cyan", no_wrap=True)
+    table.add_column("Compiler Name", style="green")
+    table.add_column("Status", justify="right")
+    count_id = 1
+    for compiler_cat in bc.compiler_modules_lookup:
+        for compiler in bc.compiler_modules_lookup[compiler_cat]:
+            table.add_row(str(count_id), compiler, "✅")
+            count_id += 1
+
+    console.print(table)
+
+    table = Table(title="Compilers Test Fail")
+    table.add_column("No.", style="cyan", no_wrap=True)
+    table.add_column("Compiler Name", style="red")
+    table.add_column("Status", justify="right")
+
+    count_id = 1
+    for compiler_cat in bc.compiler_modules_lookup_fail:
+        for compiler in bc.compiler_modules_lookup_fail[compiler_cat]:
+            table.add_row(str(count_id), compiler, "❌")
+            count_id += 1
+
+    console.print(table)
```

Comment:
add if statement saying `if table.row_count` then print the table https://rich.readthedocs.io/en/stable/reference/table.html#rich.table.Table.row_count